about.html

```diff
@@ -0,0 +1,150 @@
+          The FIFA Congress is the supreme 
+          legislative body of the International 
+          Association Football Federation 
+          (French: Fédération Internationale de Football Association), 
+          commonly known by the acronym FIFA. 
+          FIFA is the international governing 
+          body of association football, futsal and beach soccer
+        </p>
+        <p class="sub-paragraph">
+          If you have any questions related
+           to the Conference, you may email us on:<br><br><span><a href="#">sales@rcb.rw</a></span>
+             
+          </p>
+      </span>
+    </section>
+    <section class="logo-history">
+      <span>
+        <h2 class="logo-title">73rd FIFA Congress logo history</h2>
+        <hr class="line-1 line-11">
+      </span>
+        <p class="logo-details">73rd FIFA Congress logo was decided on 16 November 2022 at 09:00</p>
+        <div class="image-and-subtitle">
+            <img src="./images/page-images/73rd congress.png" alt="FIFA logo" class="logo-image">
+            <div class="more-description">
+              <hr>
+              <small class="image-description">73rd FIFA Congress 2023</small>
+              <hr>
+            </div>
+        </div>
+    </section>
+    <section class="past-events">
+      <span class="re-arrange">
+        <h2 class="event-title">See past FIFA Congress Meeting</h2>
+        <hr class="line-1 line-12">
+      </span>
+      <p class="past-event-description">Take a look at past 2 years FIFA Congress Meeting</p> 
+      <span class="past-events-photos">
+        <div class="event-image-1">
+          <h3 class="event-location">FIFA Congress Meeting at Doha</h3>
+          <p class="event-time">2022</p>
+        </div>
+        <div class="event-image-2">
+          <h3 class="event-location">FIFA Congress Meeting at Switzeland</h3>
+          <p class="event-time">2021</p>
+        </div>
+      </span>
+    </section>
+  </main>
+  <footer>
+    <div class="footer-1-container">
+      <section class="footer-section-1" id="partners">
+        <span>
+          <h2 class="footer-title">Partners</h2>
+          <hr class="line-1 line-13">
+        </span>
+        <ul class="partners-list">
+          <li>
+            <a href="#"><img src="./images/page-images/coca.png" alt="Cocacola logo" class="partners-logo"></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/page-images/adidas.png" alt="Adidas logo" class="partners-logo"></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/page-images/Qatar-Airways.png" alt="Quatar Airways logo" class="partners-logo"></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/page-images/hundai.png" alt="Hyundai logo" class="partners-logo"></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/page-images/visa.png" alt="Visa logo" class="partners-logo"></a>
+          </li>
+        </ul>
```

Comment:
- [ ] In the desktop version of the about page, kindly ensure that the partners section is hidden and not showing.

**This is the about desktop version:**

![image](https://user-images.githubusercontent.com/86318284/229807257-b284a358-798c-4590-9a6c-49b591c47566.png)

**This is how your design looks like:**

![image](https://user-images.githubusercontent.com/86318284/229807361-71fc9f5a-e30f-463d-a539-0473931284d3.png)